iroha-cpp/README.md

```diff
@@ -0,0 +1,72 @@
+# C++ library for Hyperledger Iroha
+
+
+This is a source repository for HL Iroha C++ library.
```

Comment:
This is not separate repository. So You should stard:
> This directory ...

Reply:
Updated readme file. On the other hand readme file will be updated once the source code will be finished.